docs/plugins/common.mdx

```diff
@@ -0,0 +1,709 @@
+| `parse` | (match: RegExpExecArray) => T                             | Function is ran to parse the string                  |
+| `react` | (props: T, ...rest: unknown[]) => ReactElement            | Function ran to return a React element after parsing |
+
+### `toast`
+
+Example:
+
+```tsx
+import { common } from "replugged";
+const { toast } = common;
+
+toast.toast("Toast text", toast.Kind.FAILURE, { duration: 5000 });
+```
+
+Enums:
+
+| Enum             | Description                                                       |
+| ---------------- | ----------------------------------------------------------------- |
+| `toast.Kind`     | Toast types <br />`MESSAGE` \| `SUCCESS` \| `FAILURE` \| `CUSTOM` |
+| `toast.Position` | Toast positions <br />`TOP` \| `BOTTOM`                           |
+
+Functions:
+
+| Name    | Parameters                                                                                                    | Return Type | Description                                                                  |
+| ------- | ------------------------------------------------------------------------------------------------------------- | ----------- | ---------------------------------------------------------------------------- |
+| `toast` | <Required /> `content`: string \| ReactElement \| null <br /> `kind`: `toast.Kind` <br /> `opts`: `ToastOpts` | void        | Makes a toast appear on the screen; `toast.Kind.SUCCESS` is the default type |
+
+`ToastOpts` Props:
+
+| Name        | Type             | Default              | Description                                            |
+| ----------- | ---------------- | -------------------- | ------------------------------------------------------ |
+| `component` | ReactElement     |                      | Custom toast component                                 |
+| `duration`  | number           | `3000`               | How long the toast stays on the screen in milliseconds |
+| `position`  | `toast.Position` | `toast.Position.TOP` | Toast position                                         |
+
+### `typing`
+
+Functions:
+
+| Name          | Parameters                       | Return Type | Description                                        |
+| ------------- | -------------------------------- | ----------- | -------------------------------------------------- |
+| `startTyping` | <Required /> `channelId`: string | void        | Starts your typing indicator in a specific channel |
+| `stopTyping`  | <Required /> `channelId`: string | void        | Stops your typing indicator in a specific channel  |
+
+## External Libraries
+
+### `hljs`
+
+**highlight.js** | [Documentation](https://highlightjs.org/usage/)
+
+### `lodash`
+
+**Lodash** | [Documentation](https://lodash.com/docs)
+
+### `ReactDOM`
+
+**ReactDOM** | [Documentation](https://reactjs.org/docs/react-dom.html)
+
+### `React`
+
+**React** | [Documentation](https://reactjs.org/docs/react-api.html)
```

Comment:
New react docs were released since this was written. Also reorder to put react before react dom.

```suggestion

### `React`

**React** | [Documentation](https://react.dev/)

### `ReactDOM`

**ReactDOM** | [Documentation](https://react.dev/reference/react-dom)
```